test/unit/resolveNodeUrl.test.js

```diff
@@ -0,0 +1,15 @@
+var _ = require('lodash'),
+    expect = require('chai').expect,
+    encoder = require('../..'),
+    testCases = require('../fixtures/url-resolve-list');
+
+describe('url-resolve', function () {
+    it('should resolve all URLs properly', function () {
+        _.forEach(testCases, function (test) {
```

Comment:
would love to see individual tests instead of the current fixture approach.
Probably later, create an issue to track later!